js/background.js

```diff
@@ -125,6 +125,11 @@
     'loki/loki_icon_128.png',
   ]);
 
+  // Set server-client time difference
+  (async () => {
+    await window.LokiPublicChatAPI.setClockParams();
```

Comment:
you can just call `window.LokiPublicChatAPI.setClockParams();` without wrapping it in an automatic executing function.
you also don't need to await it since the value is not needed immediately.